.circleci/config.yml

```diff
@@ -12,14 +12,7 @@ jobs:
           name: Check dependencies
           command: |
             go version
-            export PATH="$GOPATH/bin:$PATH"
-            mkdir -p "$GOPATH/bin"
-            curl --fail https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
-            dep version
-            dep status
-            case $(dep ensure 2>&1 | head -n 1) in Warning*) false ;; esac
-            dep check
-
+            go mod verify
```

Comment:
I don't think this does the same job as the previous check: 
- https://golang.org/ref/mod#go-mod-verify
- https://github.com/golang/go/issues/27348

In addition to `go mod verify`, the check should probably be something along the lines of executing `go mod vendor` again and seeing if any of the files have changed? Not sure if `go mod vendor` would always overwrite files though, so maybe even `rm -rf vendor/; go mod vendor; git add vendor; git status`?

Reply:
Right, I glossed over this thinking `go mod` would simplify it :smile:

OK, I'll see what I can do.

Reply:
@na-- Does 844578c1 cover this? See a failure [here](https://circleci.com/gh/loadimpact/k6/12148). Besides the `grep`, it seems more elegant than `rm`/`git` :)

This was an extreme example since tests wouldn't run either, but it should catch any minor package differences as well.

Reply:
hmm this check seem flaky, sorry - why not rely on git to check if the actual file contents in vendor have changed? that way you don't rely on the format of some text file or the output of some command...

and it will work much better with the `.gitignore` idea you shared below

Reply:
checking if `git status --porcelain` has any output seems good to do the job?

Reply:
Yeah, the output and order for the diff might not be stable. OK, I'll try it with `git status`.

Reply:
19d8a1f6 seems like it should do it, let me know. I want to avoid `rm`/`git add` if possible.